handbook/communication/code_of_conduct.md

```diff
@@ -0,0 +1,188 @@
+# Code of Conduct
+
+[TOC]
+
+
+## Our pledge
+
+We pledge to make participation in our company, our projects, and our community an open, welcoming, and harassment-free experience for everyone, regardless of ability, age, body size, skin color, culture, education, ethnicity, gender identity and expression, immigration status, level of experience, mental and physical ability, nationality, personal appearance, political beliefs, race, relationship or family status, religion, sex, sexual identity and orientation, or socio-economic status.
+
+We care about the safety and comfort of everyone in our community, including colleagues, contributors, and users.
+
+
+## Team member philosophy
+
+If you work at Sourcegraph or participate in our projects and community, you are important to us. If for any reason you don't feel that,
+
+<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: undefined internal link (link text: "please let us know"). Did you generate a TOC? </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
```

Comment:
I think this accidentally made it in from the docs to markdown conversion.

```suggestion
```

Reply:
It did, ugh. Good looking out